docs/source/miscellaneous/index.rst

```diff
@@ -7,3 +7,4 @@ Miscellaneous
    :titlesonly:
 
    contributing
+   doc/adr/index
```

Comment:
It seems a bit unusual that we have doc/ as a directory (see general comment on the PR). We can probably move the ADR folder to miscellaneous/ directly if we want to keep it here.